crates/pathfinder/src/state.rs

```diff
@@ -52,7 +52,8 @@ pub(crate) fn update_contract_state(
     // Update the contract state tree.
     let state_hash = global_tree
         .get(contract_address)
-        .context("Get contract state hash from global state tree")?;
+        .context("Get contract state hash from global state tree")?
+        .unwrap_or_else(|| ContractStateHash(Default::default()));
```

Comment:
I would prefer if this was `ContractStateHash::ZERO`. I realise this is probably just semantics / nitty, but it very specifically **must be** the literal zero value. And I worry that `default` just doesn't convey the intent well enough.

Reply:
Done